var/spack/repos/builtin/packages/amdfftw/package.py

```diff
@@ -0,0 +1,210 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import os
+import os.path
+import llnl.util.lang
+from spack import *
+from spack.pkg.builtin.fftw import FftwBase
+
+
+class Amdfftw(FftwBase):
+    """AMD Optimized FFTW.
+
+    FFTW is a comprehensive collection of fast C routines for
+    computing the Discrete Fourier Transform (DFT) and various special
+    cases thereof. It is an open-source implementation of the Fast
+    Fourier transform algorithm. It can compute transforms of real and
+    complex-values arrays of arbitrary size and dimension.
+    AMD Optimized FFTW is the optimized FFTW implementation targeted
+    for AMD CPUs.
+    """
+
+    _name = 'amdfftw'
+    homepage = "https://developer.amd.com/amd-aocl/fftw/"
+    url = "https://github.com/amd/amd-fftw/archive/2.2.tar.gz"
+    git = "https://github.com/amd/amd-fftw.git"
+
+    version('2.2', sha256='de9d777236fb290c335860b458131678f75aa0799c641490c644c843f0e246f8')
+    version('2.1', sha256='b58e063ddc9bb178dbc9914ae863e26ca511011357dd3ddf36fc014f5875d18c')
+    version('2.0', sha256='277b43aedbb667eda20611f52772d10c37c26b137bc5ff108554bd5f133a5e58')
+
+    variant(
```

Comment:
I'm seeing a lot of duplication in this package that shouldn't be needed since the same thing is present in `FftwBase`

Reply:
Incorporated spack community review comments under - https://github.com/spack/spack/pull/18155